wdl/transforms/new-base/src/main/scala/wdl/transforms/base/linking/expression/values/EngineFunctionEvaluators.scala

```diff
@@ -448,8 +448,10 @@ object EngineFunctionEvaluators {
                                ioFunctionSet: IoFunctionSet,
                                forCommandInstantiationOptions: Option[ForCommandInstantiationOptions])
                               (implicit expressionValueEvaluator: ValueEvaluator[ExpressionElement]): ErrorOr[EvaluatedValue[WomInteger]] = {
-      processValidatedSingleValue[WomFloat, WomInteger](a.param.evaluateValue(inputs, ioFunctionSet, forCommandInstantiationOptions)) { float =>
-        EvaluatedValue(WomInteger(math.floor(float.value).toInt), Seq.empty).validNel
+      import BigDecimal.RoundingMode.FLOOR
+      def floor(a: BigDecimal): BigDecimal = a.setScale(0, FLOOR)
+      processValidatedSingleValue[WomFloat, WomInteger](a.param.evaluateValue(inputs, ioFunctionSet, forCommandInstantiationOptions)) { bidDecimal =>
```

Comment:
This line is 143 character long, consider splitting it into two lines.

Reply:
Fixed